### ISSUE TITLE
Release 2.2.1?

### ISSUE THREAD

**Issue by hurtstotouchfire** [ffaker/ffaker#305]
Hi guys, we are adding FFaker to all of our apps primarily for https://github.com/ffaker/ffaker/pull/285. It would be great if you guys could release a new version so we don't have to point to the github repo.

**Comment by marocchino** [comment]
cc @ffaker/team

**Comment by rocknruby** [comment]
Start 🛫 

Hannes Maack

**Comment by brunowego** [comment]
Yes! Please, my BundleAudit screams because I have used `gem 'ffaker', github: 'ffaker/ffaker'`.

**Comment by rocknruby** [comment]
@marocchino are you able to release? And how about the changelog?

**Comment by marocchino** [comment]
No. I am not. I thought You can release. cause, you are in [list](https://rubygems.org/gems/ffaker).

**Comment by borourke** [comment]
I am also getting this error:

```
irb(main):001:0> FFaker::Time.between
NoMethodError: undefined method `between' for FFaker::Time:Module
```

Really needs to get fixed ASAP or taken off the list of documentation

**Comment by EmmanuelOga** [comment]
@marocchino you now have admin access to ffaker in rubygems: https://rubygems.org/profiles/marocchino

I'm sorry I can't release myself: its been a while since I'm actively involved on ffaker and I feel like I don't have enough context to perform a release.

**Comment by marocchino** [comment]
@EmmanuelOga Thank you for grant me. I will work in a day. :)

**Comment by marocchino** [comment]
@hurtstotouchfire I just released 2.3.0 version.
